drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -265,16 +464,16 @@ static int stm32_clock_control_init(struct device *dev)
 	LL_RCC_SetAPB3Prescaler(apb3_prescaler(CONFIG_CLOCK_STM32_D1PPRE));
 	LL_RCC_SetAPB4Prescaler(apb4_prescaler(CONFIG_CLOCK_STM32_D3PPRE));
 
-	/* Set PLL1 as System Clock Source */
-	LL_RCC_SetSysClkSource(LL_RCC_SYS_CLKSOURCE_PLL1);
-	while (LL_RCC_GetSysClkSource() != LL_RCC_SYS_CLKSOURCE_STATUS_PLL1) {
-	}
+	/* Set FLASH latency */
+	/* AXI clock is SYSCLK / HPRE */
+	LL_SetFlashLatency(get_bus_clock(
+							CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC,
+							CONFIG_CLOCK_STM32_HPRE));
```

Comment:
Indentation is off here

Reply:
Done, corrected, moved ```CONFIG_SYS...``` to the previous line and aligned the 2 CONFIG symbols.